simpletransformers/t5/t5_model.py

```diff
@@ -91,11 +91,18 @@ def __init__(
 
         self.results = {}
 
-        self.config = T5Config.from_pretrained(model_name, **self.args.config)
-
-        self.model = T5ForConditionalGeneration.from_pretrained(model_name, config=self.config)
+        if model_name==None:
+            self.config = config
+            self.model = T5ForConditionalGeneration(config=self.config)
```

Comment:
The `config` from args can be used instead.

```suggestion
            self.config = self.args.config
            self.model = T5ForConditionalGeneration(config=self.config)
```